SPEC.md

```diff
@@ -185,6 +188,15 @@ Plugins may define additional fields that they accept and may generate an error
 }
 ```
 
+### Version considerations
+
+CNI runtimes and plugins may support multiple versions of the protocol. To indicate that a particular configuration supports
```

Comment:
@squeed maybe "CNI runtimes, plugins, and configurations may each support multiple versions of the CNI specification independently. Plugins indicate supported CNI specification versions through the reply to the `VERSION` command, while network configurations may indicate multiple supported CNI specification versions through the `cniVersions` field.

Runtimes are thus able to query both plugins and configurations and select a commonly supported CNI specification version for the CNI request. The runtime MUST select the highest supplied version that it supports. If both `cniVersion` and `cniVersions` are supplied, the runtime MUST select the highest of the two.